src/app/components/sections/Feature.tsx

```diff
@@ -0,0 +1,65 @@
+'use client'
+
+import { useTranslations } from "next-intl";
+import { FeatureCard } from "@/src/app/components/ui/FeatureCard";
+import { motion } from "framer-motion";
+
+const container = {
+    hidden: {},
+    show: {
+        transition: {
+            staggerChildren: 0.5,
+        },
+    },
+};
+
+const item = {
+    hidden: { opacity: 0, y: 20 },
+    show: { opacity: 1, y: 0 },
+};
+
+export function Feature() {
+    const t = useTranslations('features');
+
+    return (
+        <div className="flex flex-col bg-amber-50  m-8 sm:m-10 lg:m-20 h-[10000px]" >
+            <h2 className="text-4xl sm:text-5xl lg:text-6xl font-bold text-center mb-12 text-black">
+                {t('title')}
+            </h2>
+
+            <div className="flex flex-col sm:gap-16 gap-8">
+                {/* Première ligne de cartes */}
+                <motion.div
+                    className="flex flex-col sm:flex-row justify-center items-center sm:gap-16 gap-8"
+                    variants={container}
+                    initial="hidden"
+                    whileInView="show"
+                    viewport={{ once: true, amount: 0.3 }}
+                >
+                    <motion.div variants={item} >
+                        <FeatureCard titleKey="localisation" subtitleKey="subtitle_localisation" />
+                    </motion.div>
+                    <motion.div variants={item} >
+                        <FeatureCard titleKey="sport" subtitleKey="subtitle_sport" />
+                    </motion.div>
+                </motion.div>
+
+                {/* Deuxième ligne de cartes */}
+                <motion.div
+                    className="flex flex-col sm:flex-row justify-center items-center sm:gap-16 gap-8"
+                    variants={container}
+                    initial="hidden"
+                    whileInView="show"
+                    viewport={{ once: true, amount: 0.3 }}
+                >
+                    <motion.div variants={item} >
+                        <FeatureCard titleKey="share" subtitleKey="subtitle_share" />
+                    </motion.div>
+                    <motion.div variants={item} >
+                        <FeatureCard titleKey="social" subtitleKey="subtitle_social" />
+                    </motion.div>
+                </motion.div>
```

Comment:
comme tu as deux blocs de div identique, tu peux créer un tableau d'entrée avec les differentes propriétés et boucler dessus pour afficher les 4 featureCard.

Par exemple :

```suggestion
                const featureRows = [
  [
    { titleKey: "localisation", subtitleKey: "subtitle_localisation" },
    { titleKey: "sport", subtitleKey: "subtitle_sport" },
  ],
  [
    { titleKey: "share", subtitleKey: "subtitle_share" },
    { titleKey: "social", subtitleKey: "subtitle_social" },
  ],
];

      {featureRows.map((row, rowIndex) => (
        <motion.div
          key={rowIndex}
          className="flex flex-col sm:flex-row justify-center items-center sm:gap-16 gap-8"
          variants={container}
          initial="hidden"
          whileInView="show"
          viewport={{ once: true, amount: 0.3 }}
        >
          {row.map((feature, index) => (
            <motion.div key={index} variants={item}>
              <FeatureCard
                titleKey={feature.titleKey}
                subtitleKey={feature.subtitleKey}
              />
            </motion.div>
          ))}
        </motion.div>
      ))}
```